subprojects/gradle/cd/build.gradle.kts

```diff
@@ -11,7 +11,6 @@ dependencies {
     implementation(project(":subprojects:gradle:android"))
     implementation(project(":subprojects:gradle:impact-shared"))
     implementation(project(":subprojects:gradle:impact"))
-    implementation(project(":subprojects:gradle:docs"))
```

Comment:
this change doesn't relate to the issue
it would be easier to make in a separate PR

Reply:
but it blocks kubernetes upgrade

Reply:
I see